microsoft-edge/devtools-guide-chromium/whats-new/2024/06/devtools-126.md

```diff
@@ -0,0 +1,69 @@
+---
+title: What's New in DevTools (Microsoft Edge 126)
+description: "DevTools displays browser extension icons instead of a blank square. Font for DevTools UI in Chinese has changed back to the accustomed font. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 06/13/2024
+---
+# What's New in DevTools (Microsoft Edge 126)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Icons for browser extensions are displayed instead of blank squares
+
+<!-- Subtitle: The DevTools extensions that you install will display their icons so they're easily identifiable in the Activity Bar.-->
+
+In previous versions of Microsoft Edge, there was a bug where some browser extensions were missing an icon when opened in DevTools.  For example, when the React extension was activated, the icon for the extension in the **Activity Bar** was displayed as a grey box.
+
+<!-- todo: add screenshot if can get one from Stable 124 -->
+
+This has been fixed; icons for browser extensions are correctly displayed in the **Activity Bar**.
+
+![Display React extension icons](./devtools-126-images/react-extension-icon.png)
```

Comment:
is it correct that there are two icons for React extension: one big, then one small?
@leahmsft 

rendered article:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/06/devtools-126?branch=pr-en-us-3192